words.html

```diff
@@ -403,447 +407,445 @@
+ale na koniec, ich chciwość stała się przyczyną ich upadku.
 </p>
 <p id="evo_9_tft">
-So, in the long run, you were right - <span class="tft">Copycat</span> wins!
-<span class="all_d">Always Cheat</span> may have won in the short run,
-but its exploitativeness was its downfall.
+A więc, na dłuższa metę, miałeś rację — <span class="tft">Papugi</span> wygrywają!
+<span class="all_d">Notoryczni Oszuści</span> mogli na początku zwyciężać,
+ale ich chciwość stała się przyczyną ich upadku.<br><br>
 </p>
 <p id="evo_9_end">
-This reminds me of a quote:
+To przypomina mi o takim cytacie:
 <br><br>
-<i>"We are punished by our sins, not for them."</i><br>
+<i>"Jesteśmy karani nie za nasze grzechy, lecz przez nie."</i></i><br>
 ~ Elbert Hubbard
 </p>
 <p id="evo_9_btn">
-(oh, and by the way...)
+(aha, tak przy okazji...)
 </p>
 
 <p id="evo_10">
-(...this result is similar even if we put
-<span class="grudge">Grudger</span> &amp; <span class="prober">Detective</span> back in:)
+(...wynik jest podobny nawet jeśli zaprosimy do gry
+<span class="grudge">Mściciela</span> i <span class="prober">Detektywa</span>:)
 </p>
 
 <p id="evo_autoplay">
-start the evolution process!
+rozpocznij proces ewolucji!
 </p>
 <p id="evo_autoplay_stop">
-stop the evolution process
+zatrzymaj proces ewolucji
 </p>
 
 <p id="evo_10_followup">
-(Note: sometimes, a few <span class="grudge">Grudgers</span> may stick around,
-because when all players except <span class="grudge">Grudger</span> &amp; <span class="tft">Copycat</span> 
-are eliminated, the two tie.)
+(Uwaga: czasami kilku <span class="grudge">Mścicieli</span> może przetrwać,
+ponieważ gdy wszyscy pozostali gracze zostaną wyeliminowani, tych dwóch remisuje.)
 <br><br>
-So, it seems the math of game theory is telling us something:
-that <span class="tft">Copycat's</span> philosophy,
-"Do unto others as you would have them do unto you", may be not just a 
-<i>moral</i> truth, but also a <i>mathematical</i> truth.
-However...
+No to wygląda na to, że matematyka teorii gier ma nam coś do powiedzenia:
+filozofia <span class="tft">Papug</span>,
+"Nie czyń drugiemu co tobie niemiłe", może być nie tylko
+prawdą <i>moralną</i>, ale też prawdą <i>matematyczną</i>.
+Jednakże...
 </p>
 <p id="evo_10_btn">
-...there's a problem: &rarr;
+...jest pewien problem: &rarr;
 </p>
 
 <p id="evo_11">
-Look around. The world's full of total jerkwads.
+Rozejrzyj się. Świat pełen jest drani.
 <br><br>
-If <span class="tft">Copycat</span> is the strategy in this repeated game of trust that's so powerful --
-that even soldiers in World War I trenches independently "evolved" a similar strategy, called "live and let live" -- 
-why, then, are there so many un-trusting, un-trustworthy people?
-What's causing our epidemic of un-trust?
+Jeśli <span class="tft">Papuga</span> jest strategią, która w tej wielokrotnej grze zaufania jest tak potężna,
+że nawet żołnierze w okopach Pierwszej Wojny Światowej niezależnie wykształcili podobną strategię —
```

Comment:
Dla mnie to bez sensu, bo to jest jedno zdanie, a w pierwszej jego części jest o strategii, więc wiadomo, że wykształcili podobną (strategię)